x-pack/plugins/uptime/public/components/overview/alerts/alerts_containers/alert_monitor_status.tsx

```diff
@@ -27,19 +37,53 @@ export const AlertMonitorStatus: React.FC<Props> = ({
   numTimes,
   setAlertParams,
   timerange,
+  alertParams,
 }) => {
-  const { filters, locations } = useSelector(selectMonitorStatusAlert);
-  const searchText = useSelector(searchTextSelector);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(
+      fetchOverviewFilters({
+        dateRangeStart: 'now-15m',
```

Comment:
i think we should set it to default for last 24 hours, last 15mins is a too small time period